script/Ciel/Day 14/14_01_CIEL14_1C.txt

```diff
@@ -756,67 +836,77 @@ If I kill you, then I won't be able to drink your blood. If you make a contract
 [sha:2af51cafd1b12586aa11979f05a313c03fc5470d]{
 -- Page 195, Offset 37583.
 -- 「……それはいいけどさ。俺がおまえに従うと仮定して、ロアをどうするつもりなんだ。
-"...There's nothing wrong with that. Even if I were to follow what you said, what do you plan to do about Roa?
+"There's nothing wrong with that...
+ Let's say I do agree. What are you going to do about Roa?
 }
 [sha:2bd37f673f005df87b72ecd51ab4aedfa680728d]{
 -- Page 195, Offset 37584.
 -- おまえは俺を殺さないっていうけど、俺が生きているかぎりロアは生き続けるんだぞ」
-You said that you wouldn't kill me, but as long as I'm alive, then Roa will continue living as well."
+You said you wouldn't kill me, but as long as I'm alive, then Roa will be too."
 }
 [sha:2eb2fa805e716c63573b4d190153bee9d792b220]{
 -- Page 196, Offset 37585.
 -- 「希薄になったロアは放っておいてもかまわないわ。
-"I don't mind leaving Roa alone if he's weak.
+"I don't mind leaving Roa alone now that he's been diluted.
 }
 [sha:938bdfbeb901dedc96887ddfd96dba016b844647]{
 -- Page 196, Offset 37586.
 -- ……そうね、だから貴方を見逃してあげてもよかった」
-...Yeah. That's why I should have let you go as well."
+Yeah... with him this weak, I could probably have just left you alone as well."
 }
 [sha:ea9a25de5d21a35dd44c1a7d9f6d0417a9222701]{
 -- Page 197, Offset 37587.
 -- 「なんだ。遠慮しないで見逃せよ。俺なんかにかまってないでさ、早く故郷に帰ってパパを安心させてやれ」
-"What's up with that? Then you should just let me go. You shouldn't bother with me, and just go back home and not worry your father."
+"Oh, well why didn't you say so.
+ By all means, leave me alone then.
+ Just forget about me and head on back to your hometown before your father starts to worry where you've gone."
 }
 [sha:67be57bf1d15664eed7c2b7eeda59547958651b3]{
 -- Page 198, Offset 37588.
 -- 「でもね。もともと自分の力なんだから取り戻そうとするのは当然でしょう？
-"But you know what? That power does belong to me after all, so it's only natural that I take it back, right?
+"But you see... It's my power, so it's only natural that I should want to take it back, right?
 }
 [sha:4e420c24f2fc8620559cdb987d68bafd22327fcb]{
 -- Page 198, Offset 37589.
 -- それに、なにより―――」
-And besides, more than anything―――"
+And, there's something I want even more―――"
 }
 [sha:8171de4b8996b8e79754ad031512aff74ec328da]{
 -- Page 198, Offset 37590.
 -- ぎり、と。
-She holds me tighter.
+Her grip tightens.
 }
 [sha:007615076231cb114efe67c47f87dc00ac3b9284]{
 -- Page 198, Offset 37591.
 -- 両肩にかかる体重が、重くなる。
-The weight on my shoulders grow heavier.
+The weight on my shoulders grows heavier.
 }
 [sha:a2f449b6be8d7e576b5914af9b8380f60f7e2d79]{
 -- Page 199, Offset 37592.
 -- 「……わたしは貴方を殺してロアを眠らせるより、このまま志貴を取りこみたい。殺して無くしてしまうより、もっと一緒に過ごしたい。前みたいに、志貴の声を聞いていたい」
-"...Rather than killing you and letting Roa sleep, I want to take you the way you are now. Rather than killing you and losing you, I want to spend more time with you. I want to hear your voice just like before."
+"Instead of killing you and forcing Roa to sleep,
+ I want to capture you as you are.
+ Instead of killing you and losing you,
+ I want to spend more time together.
+ I want to hear you talk with me like you used to."
```

Comment:
Just so these are all aligned AND it fits with the previous dialogue.
>I want to capture you as you are instead of killing you and forcing Roa to sleep.
>I want to spend more time together instead of killing you and losing you forever.
>I want to hear you talk with me like you used to.

Maybe find another word for one of the "killing". Might be a bit TOO much repetition?